docs/advanced-test-features/test-configuration.md

```diff
@@ -34,7 +34,7 @@ overrides any child runnable's configuration.
 [`Configurable` in the API
 docs](https://inferno-framework.github.io/inferno-core/docs/Inferno/DSL/Configurable.html)
 
-#### Renaming Inputs, Outputs, and Requests
+<h3 data-toc-skip class="h4">Renaming Inputs, Outputs, and Requests</h3>
```

Comment:
```suggestion
### Renaming Inputs, Outputs, and Requests
{:toc-skip}
```
See comment on `community/index.md` regarding use of Kramdown attribute list definitions to avoid inline HTML.